api/api/src/main/scala/uk/ac/wellcome/platform/api/models/WorkFilter.scala

```diff
@@ -16,3 +18,6 @@ case object WorkTypeFilter {
   def apply(workTypeId: String): WorkTypeFilter =
     WorkTypeFilter(workTypeIds = Seq(workTypeId))
 }
+
+case class DateRangeFilter(fromDate: Option[DateTime], toDate: Option[DateTime])
```

Comment:
Should these be non-optional?

Reply:
I was thinking for cases where the user wants e.g any work created before the 20th century, where they might enter something like this:

```
/api/works/?date_to=1900
```

It makes sense to me to make either `fromDate` or `toDate` optional because of this (note this implementation is still unfinished). Thoughts?

Reply:
Ah, I see, makes perfect sense and adds value.